region/maridia/outer.json

```diff
@@ -1240,19 +1296,87 @@
                         "fromNode": 1,
                         "inRoomPath": [1, 8],
                         "framesRemaining": 120,
-                        "shinesparkFrames": 75
+                        "shinesparkFrames": 77
                       }},
                       {"canComeInCharged": {
                         "fromNode": 2,
                         "inRoomPath": [2, 8],
                         "framesRemaining": 180,
-                        "shinesparkFrames": 75
+                        "shinesparkFrames": 77
                       }}
                     ]}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": []
+                    }
                   ]
                 }
               ],
-              "devNote": "The short charge actually has 2 steep up tiles, but one is against a wall so it was excluded"
+              "devNote": [
+                "The short charge actually has 2 steep up tiles, but one is against a wall so it was excluded",
+                "FIXME: Add a midair version requiring 0 tanks"
+              ]
+            },
+            {
+              "id": 9,
+              "strats": [
+                {
+                  "name": "Shinespark with Suit",
+                  "notable": false,
+                  "requires": [
+                    "Gravity",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 17,
+                        "steepUpTiles": 1,
+                        "shinesparkFrames": 76,
+                        "openEnd": 0
+                      }},
+                      {"canComeInCharged": {
+                        "fromNode": 1,
+                        "inRoomPath": [1, 8],
+                        "framesRemaining": 40,
+                        "shinesparkFrames": 76
+                      }},
+                      {"canComeInCharged": {
+                        "fromNode": 2,
+                        "inRoomPath": [2, 8],
+                        "framesRemaining": 70,
+                        "shinesparkFrames": 76
+                      }}
+                    ]}
+                  ],
+                  "note": "To avoid breaking the speed blocks, align Samus on the first plant, just to the left of the door."
+                },
+                {
+                  "name": "Suitless Shinespark",
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    {"or": [
+                      {"canComeInCharged": {
+                        "fromNode": 1,
+                        "inRoomPath": [1, 8],
+                        "framesRemaining": 60,
```

Comment:
This one could go down to 20, but 30 is a safer guess.  And `canShinesparkMovement` since you jumped through this door.